networkx/algorithms/centrality/current_flow_betweenness.py

```diff
@@ -64,11 +66,18 @@ def approximate_current_flow_betweenness_centrality(
        "cg" (uses least memory).
 
     epsilon: float
-        Absolute error tolerance.
+        Absolute error tolerance. Note that smaller values of epsilon lead to
+        higher numbers of sample pairs (`k`) and thus more computation time. The number
+        of sample pairs is approximately `(c/epsilon)^2 * log(n)` where `n` is the
+        number of nodes.
 
     kmax: int
        Maximum number of sample node pairs to use for approximation.
 
+    sample_weight: float (default=1)
```

Comment:
It'd be good to make this change for the other parameters as well!

```suggestion
    sample_weight : float (default=1)
```